CHANGELOG.md

```diff
@@ -32,3 +33,5 @@
 ## 0.1.0
 
 - First release
+
+[@vergenzt]: https://github.com/vergenzt
```

Comment:
Sorry about that. Last repo I contributed to did this and I didn't think about it.